src/storage/btree_load.c

```diff
@@ -5094,9 +5081,19 @@ index_builder_loader_task::execute (cubthread::entry &thread_ref)
       return;
     }
 
-  for (auto &key_oid : m_keys_oids)
+  PERF_UTIME_TRACKER time_insert_task = PERF_UTIME_TRACKER_INITIALIZER;
+  PERF_UTIME_TRACKER_START (&thread_ref, &time_insert_task);
+
+  PERF_UTIME_TRACKER time_prepare_task = PERF_UTIME_TRACKER_INITIALIZER;
+  PERF_UTIME_TRACKER_START (&thread_ref, &time_prepare_task);
+
+  m_insert_list.prepare_list ();
+
+  PERF_UTIME_TRACKER_TIME (&thread_ref, &time_prepare_task, PSTAT_BT_ONLINE_PREPARE_TASK);
+
```

Comment:
shouldn't timer for PSTAT_BT_ONLINE_INSERT_TASK start after prepare? if so, only one tracker is needed.

Reply:
insert task time = prepare task time + execution time
execution time is not measure separately
prepare task time = mostly time required for sorting of chunk.